index.bs

```diff
@@ -485,6 +510,85 @@ spec:csp3; type:grammar; text:base64-value
   failed resource with a different one.
 
   <!-- ####################################################################### -->
+  
+  ## Integrity-Policy ## {#integrity-policy-section}
+  The `Integrity-Policy` and `Integrity-Policy-Report-Only` HTTP headers enable a document to
+  enforce a policy regarding the integrity metadata requirements on all the subresources it
+  loads of certain <a>destinations</a>.
+
+  The headers' value is a <a>Dictionary</a> [[RFC9651]], with every member-value being an
+  <a>inner list</a> of <a>tokens</a>.
+  
+  An <dfn>integrity policy struct</dfn>, is a <a>struct</a> that contains the following:
+    * <dfn>enforcement</dfn>, a string which is the empty string or "block". Initially "block".
+    * <dfn>source</dfn>, a string which is the empty string or "inline". Initially "inline".
+    * <dfn>destination</dfn>, a string which is the empty string or "script". Initially the empty string.
+    * <dfn>endpoints</dfn>, a <a>list</a> of strings, initially empty.
+
+  When <dfn>processing an integrity policy</dfn>, with |input|, do the following:
+  1) Let |integrity policy| be a new <a>integrity policy struct</a>.
+  1) Let |dictionary| be the result of <a>parsing a dictionary</a> with |input|.
+  1) If |dictionary|['enforcement'] exists:
+    1) If its value is "block", set |integrity policy|'s <a>enforcement</a> to "block".
+    1) Otherwise, set |integrity policy|'s <a>enforcement</a> to the empty string.
+  1) If |dictionary|['sources'] exists:
+    1) If its value <a>contains</a> "inline", set |integrity policy|'s <a>source</a> to "inline".
```

Comment:
The `contains` you are using here is linking to https://fetch.spec.whatwg.org/#header-list-contains. That does not appear to be correct?

Reply:
Indeed, made the refs explicit!